apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/components/template-selector/shared.scss

```diff
@@ -0,0 +1,24 @@
+$wp-org-sidebar-reduced: 36px;
+$wp-org-sidebar-full: 160px;
```

Comment:
Are these meant to replace some existing variables? This seems dangerous, see my other comment about global variables and namespaces.

Reply:
I think they're just used by SPT for some calculations.

We could use these upstream vars though: https://github.com/WordPress/gutenberg/blob/fdd69535ecbec018bfabad783c890de3d72b7e07/packages/base-styles/_variables.scss#L30-L32

Reply:
Pushed f4adda3 and ba960df to that effect.